XRTK.Oculus/Packages/com.xrtk.oculus/Runtime/OculusPlatform.cs

```diff
@@ -0,0 +1,26 @@
+﻿using XRTK.Definitions.Platforms;
+
+namespace XRTK.Oculus
+{
+    public class OculusPlatform : BasePlatform
+    {
+        /// <inheritdoc />
+        public override bool IsAvailable => !UnityEngine.Application.isEditor && OculusApi.Version != null && OculusApi.Initialized;
```

Comment:
OculusApi.Version will never be null. The least it will be `new Version(0,0,0)`.

Reply:
In fact, I think I'd omit the version check here, because it's not relevant to whether the platform is available. `OculusApi.Initialized` should be sufficient.